docs/release/release_0_4_8.md

```diff
@@ -0,0 +1,157 @@
+# napari 0.4.8
+
+We're happy to announce the release of napari 0.4.8!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+## Highlights
+
+- Add Hookspec for providing sample data (#2483)
+- Text overlay visual (#2595)
+
+## New Features
+
+- Add highlight widget to preferences dialog (#2435)
+- PR: Add interface language selection to preferences (#2466)
+- Add Hookspec for providing sample data (#2483)
+- Add ability to run file as plugin (#2503)
+- Add `layer.source` attribute to track layer provenance (#2518)
+- Add button to drop into debugger in the traceback viewer. (#2534)
+- PR: Add initial welcome screen on canvas (#2542)
+- Text overlay visual (#2595)
+
+## Improvements
+
+- Add stretch to vertical dock widgets (#2154)
+- Use new selection model on existing `LayerList` (#2441)
+- Add bbox annotator example (#2446)
+- Save last working directory (#2467)
+- add name to dock widget titlebar (#2471)
+- Add Qt`ListModel` and `ListView` for `EventedList` (#2486)
+- New new qt layerlist (#2493)
+- Move plugin sorter (#2501)
+- Add ColorManager to Vectors (#2512)
+- PR: Enhance translation methods (#2517)
+- PR: Add typing to schema_version (#2536)
+- PR: Add initial restart implementation (#2540)
+- Add data setter for `surface` layers (#2544)
+- Extract shortcut into their own object. (#2554)
+- example of using magicgui with thread_worker (#2577)
+- Add example tying slider change to point properties change (#2582)
+- Range of label spinbox is more dtype-aware (#2597)
+- PR: Add generic name to unnamed dockwidgets (#2604)
+- PR: Hide ipy interactive option (#2605)
+- PR: Add option to save state separate from geometry (#2606)
+- QtLargeIntSpinbox for label controls (#2608)
+- Support varying number of dimensions during labels painting (#2609)
```

Comment:
Ok I'm biased but I'd consider highlighting this. 😂

Reply:
done